src/storage-types/src/parameters.proto

```diff
@@ -47,9 +48,16 @@ message ProtoPgSourceTcpTimeouts {
     optional mz_proto.ProtoDuration tcp_user_timeout = 5;
 }
 
+message ProtoKafkaSourceTcpTimeouts {
```

Comment:
```suggestion
message ProtoKafkaSourceTimeouts {
```

Only the `socket_timeout` and `socket_connection_setup_timeout` are related to `Tcp`, the rest are application layer timeouts

Reply:
its actually sources and sinks so ill just name it `KafkaTimeouts`